report/Sprint_1/Application.tex

```diff
@@ -0,0 +1,92 @@
+\section{Application}
+During the first sprint, we started development on the application.
+The following features were implemented:
+\begin{itemize}
+    \item Login screen
+    \item Application selection screen
+    \item Create application screen
+    \item Create user screen
+    \item Task dashboard screen
+    \item Create task screen
+\end{itemize}
+
+In this sprint, some of the functionality was mocked, because they were yet to be developed on the backend.
+This was however limited to task functionality, where the rest was actual data transmitted from the API.
+
+\subsection{Design}
+\label{sprint_1_design}
+In the start of this sprint, some views were designed to give an idea of how the application should look.
+The designs relevant to the features developed in this sprint.
+
+The screens were designed to have a generic look, and feel familiar.
+The tasks screen, which can be seen on ~\autoref{task_dashboard_screen}, was designed to look like a kanban board \cite{kanbanBoard}.
+This was done because Kanban boards are a common way to organise tasks.
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth]{Sprint_1/images/login_screen.png}
+    \caption{Login screen design.}
+    \label{login_screen_design}
+\end{figure}
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth]{Sprint_1/images/app_selection_screen.png}
+    \caption{Application selection screen design.}
+    \label{app_selection_screen}
+
+\end{figure}
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth]{Sprint_1/images/create_task_screen.png}
+    \caption{Create task screen design.}
+    \label{create_task_screen}
+
+\end{figure}
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth]{Sprint_1/images/task_dashboard_screen.png}
+    \caption{Task dashboard screen design.}
+    \label{task_dashboard_screen}
+\end{figure}
```

Comment:
Synes billederne er lidt for store. Enten gør det alle sammen lidt mindre, eller kun tag nogle af dem, og smid resten i appendix.
Derudover så kunne du godt forklare billederne lidt mere, i forhold til hvad vi har tænkt med det.

Reply:
Jeg har gjort dem mindre ved at sætte dem sammen 2 og 2